src/utils/recipient.js

```diff
@@ -85,12 +85,17 @@ export const getRecipientsIds = (message: Message, ownEmail?: string): string =>
     throw new Error('getRecipientsIds: expected PM, got stream message');
   }
   const recipients = message.display_recipient;
-  return recipients.length === 2
-    ? recipients.filter(r => r.email !== ownEmail)[0].id.toString()
-    : recipients
-        .map(s => s.id)
-        .sort((a, b) => a - b)
-        .join(',');
+  if (recipients.length === 2) {
+    if (ownEmail === undefined) {
+      throw new Error('getRecipientsIds: got 1:1 PM, but ownEmail omitted');
+    }
+    return recipients.filter(r => r.email !== ownEmail)[0].id.toString();
```

Comment:
I mean, it's certainly better than the status quo in which there's no check at all, but... :nauseated_face:

If you can't reasonably split this into two separate functions, can you at least give it an explanatory (and, presumably, abjuratory) docstring? Ideally with a `TODO:` with a suggested path forward. (...and possibly rename it not to have a plural as a [noun adjunct](https://en.wikipedia.org/wiki/Noun_adjunct).)

Unrelatedly, the summary on this commit is a little too long. (GitHub wraps them after 72 characters.)

Reply:
Yeah, there's still further improvements this function needs. As I mentioned above:

> I have some draft other changes, including to give `getRecipientsIds` a better name (turns out what it's really about is specifically for the "unread messages" data structure) and some jsdoc

---

> Unrelatedly, the summary on this commit is a little too long. (GitHub wraps them after 72 characters.)

What we say [in our docs](https://zulip.readthedocs.io/en/latest/contributing/version-control.html#commit-messages)￼, following some discussion in the last few months, is that the summary:
> is kept short (max 76 characters, ideally less), while concisely explaining what the commit does

This is 75ch, so I guess it's just under the wire.